packages/epo-widget-lib/src/atomic/Blinker/Images/Images.tsx

```diff
@@ -0,0 +1,51 @@
+import { FunctionComponent, useEffect, useState } from "react";
+import { CircularLoader } from "@rubin-epo/epo-react-lib";
+import * as Styled from "./styles";
+import BlinkerImage from "../Image/Image";
+import { BlinkerProps } from "../Blinker";
+
+interface ImagesProps extends Pick<BlinkerProps, "images" | "activeIndex"> {
+  className?: string;
+}
+
+const Images: FunctionComponent<ImagesProps> = ({
+  images = [],
+  activeIndex,
+  className,
+}) => {
+  const [imagesLoaded, setImagesLoaded] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const loadCallback = () => {
+    setImagesLoaded((count) => count + 1);
+  };
+
+  useEffect(() => {
+    setIsLoading(imagesLoaded !== images.length);
```

Comment:
there was a reset method dependent on the `images` prop changing in the original component, however what ended up happening was that if you removed a value from the array React would simply drop it from the render but not touch the other images, so it would just get stuck on a loading screen forever.